owncloudApp/src/main/java/com/owncloud/android/presentation/authentication/oauth/OAuthUtils.kt

```diff
@@ -113,7 +117,10 @@ class OAuthUtils {
                 appendQueryParameter(QUERY_PARAMETER_CODE_CHALLENGE, codeChallenge)
                 appendQueryParameter(QUERY_PARAMETER_CODE_CHALLENGE_METHOD, CODE_CHALLENGE_METHOD)
                 appendQueryParameter(QUERY_PARAMETER_STATE, state)
-                if (!username.isNullOrEmpty()) appendQueryParameter(QUERY_PARAMETER_USERNAME, username)
+                if (sendLoginHintAndUser && !username.isNullOrEmpty()) {
+                    appendQueryParameter(QUERY_PARAMETER_USER, username)
```

Comment:
why have you changed the name from `QUERY_PARAMETER_USERNAME` to `QUERY_PARAMETER_USER`?. Why haven't you also changed it in the `username` variable?

Reply:
Because the name of the parameter is `user` and not `username` as you can see in the definition of that constant. I kept `username` like that because otherwise it can be confused with an user (`OCAccount`) object, in this case it's just the name. So, this way we keep consistency for both network request (keeping the same name as the parameter) and business logic (not confusing it with the user object)